### ISSUE TITLE
Must have cmake

### ISSUE THREAD

**Issue by darthnithin** [micheleriva/face-detection#1]
Didn't work until I manually installed cmake on my windows machine via https://cmake.org/download/ though this is not listed anywhere as a requirement nor was autoinstalled.

**Comment by micheleriva** [comment]
@darthnithin thanks for noticing this! I'm updating the package asap.

**Comment by darthnithin** [comment]
Yes while cmake helped get further in the installation i don't believe I was able to get it fully running unfortunately.
